tests/unittests/tests-ipv6_addr/tests-ipv6_addr.c

```diff
@@ -574,6 +574,26 @@ static void test_ipv6_addr_init_prefix(void)
     TEST_ASSERT_EQUAL_INT(true, ipv6_addr_equal(&a, &c));
 }
 
+static void test_ipv6_addr_init_iid(void)
+{
+    ipv6_addr_t a = { {
+            0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
+            0x08, 0x09, 0x0a, 0x0b, 0xff, 0x0d, 0x0e, 0x0f
+        }
+    };
+
+    const uint8_t iid[] = { 0x13, 0x02, 0x01, 0x00 };
+
+    ipv6_addr_t c =  { {
+            0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07,
+            0x08, 0x09, 0x0a, 0x0b, 0x93, 0x02, 0x01, 0x00
+        }
+    };
+
+    ipv6_addr_init_iid(&a, iid, 31);
```

Comment:
it breaks for the NULL cases. bits > 128 will be limited to 128. Using NULL as input parameters makes no sense here. I don't check for this condition within this function and outsource this responsibility to the user of this function. Like it was done with `ipv6_addr_init_prefix()`